cheat-sheet.md

```diff
@@ -347,4 +347,14 @@
   		      agent.mute();
   		  }
   	}
-```
\ No newline at end of file
+```
+
+## Get contact is ScreenRecording Enabled
```

Comment:
Update the text to be something like "Check if screen recording is enabled on a contact"

Reply:
thx, renamed,